src/strategies/LidoLevL2.sol

```diff
@@ -0,0 +1,235 @@
+    /*//////////////////////////////////////////////////////////////
+                           VALUATION
+    //////////////////////////////////////////////////////////////*/
+
+    function totalLockedValue() public view override returns (uint256) {
+        uint256 aaveDebt = debtToken.balanceOf(address(this));
+        uint256 aaveCollateral = _wstEthToEth(aToken.balanceOf(address(this)));
+
+        return aaveCollateral - aaveDebt;
+    }
+
+    AggregatorV3Interface public constant WSTETH_ETH_FEED =
+        AggregatorV3Interface(0x806b4Ac04501c29769051e42783cF04dCE41440b);
+
+    /*//////////////////////////////////////////////////////////////
+                                  STAKED ETHER
+    //////////////////////////////////////////////////////////////*/
+
+    IWETH public constant WETH = IWETH(payable(0x4200000000000000000000000000000000000006));
+    IWSTETH public constant WSTETH = IWSTETH(0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22);
+
+    function _getEthToWstEthRatio() internal view returns (uint256) {
+        (uint80 roundId, int256 price,, uint256 timestamp, uint80 answeredInRound) = WSTETH_ETH_FEED.latestRoundData();
+        require(price > 0, "LidoLevL2: price <= 0");
+        require(answeredInRound >= roundId, "LidoLevL2: stale data");
+        require(timestamp != 0, "LidoLevL2: round not done");
+        return uint256(price);
+    }
+
+    function _wstEthToEth(uint256 amountWstEth) internal view returns (uint256) {
+        return amountWstEth.mulWadDown(_getEthToWstEthRatio());
+    }
+
+    function _ethToWstEth(uint256 amountEth) internal view returns (uint256) {
+        // This is (ETH * 1e18) / price. The price feed gives the ETH/wstETH ratio, and we divide by price to get amount of wstETH.
+        // `divWad` is used because both both ETH and the price feed have 18 decimals (the decimals are retained after division).
+        return amountEth.divWadDown(_getEthToWstEthRatio());
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                                 TRADES
+    //////////////////////////////////////////////////////////////*/
+    ICurvePool public constant CURVE = ICurvePool(0x11C1fBd4b3De66bC0565779b35171a6CF3E71f59);
+    uint256 public slippageBps = 60; // 0.6%
+
+    function setSlippageBps(uint256 _slippageBps) external onlyRole(STRATEGIST_ROLE) {
+        slippageBps = _slippageBps;
+    }
+    /*//////////////////////////////////////////////////////////////
+                                  AAVE
+    //////////////////////////////////////////////////////////////*/
+
+    /// @notice The leverage factor of the position multiplied by 100. E.g. 150 would be 1.5x leverage.
+    uint256 public leverage = 992; // 9.92x
+
+    function setLeverage(uint256 _leverage) external onlyRole(STRATEGIST_ROLE) {
+        leverage = _leverage;
+    }
+
+    /// @notice The percentage of the collateral to borrow from AAVE in bps.
+    uint256 public borrowBps = 8999; // 89.99%
```

Comment:
is it required? No usage.

Reply:
Good catch. This is used to determine how much wETH to borrow during deposits.